multiqc/multiqc.py

```diff
@@ -438,6 +449,8 @@ def run(
         config.exclude_modules = exclude
     if profile_runtime:
         config.profile_runtime = True
+    if custom_css_files is not None and len(custom_css_files) > 0:
```

Comment:
I have a feeling that if a click argument accepts multiple, it returns an empty list of not specified. So you can probably ditch this line if that's true.

But worth checking as this is off the top of my head 😅

Reply:
Will test!